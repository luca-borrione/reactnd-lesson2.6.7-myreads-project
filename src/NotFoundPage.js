
import React from 'react'

const NotFoundPage = props => {

	return (
		<div className="notfound-page-wrap">
			<h1>404</h1>
			<h2>Not Found</h2>
		</div>
	);

};

export default NotFoundPage;