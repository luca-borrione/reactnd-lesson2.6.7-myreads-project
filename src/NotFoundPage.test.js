import React from 'react';
import ReactDOM from 'react-dom';
import TestRenderer from 'react-test-renderer';
import NotFoundPage from './NotFoundPage';

describe('NotFoundPage', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<NotFoundPage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });


  it('renders correctly', () => {
    const tree = TestRenderer
      .create(<NotFoundPage />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
