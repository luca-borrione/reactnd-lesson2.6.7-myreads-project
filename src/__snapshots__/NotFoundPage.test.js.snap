// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders correctly 1`] = `
<div
  className="notfound-page-wrap"
>
  <h1>
    404
  </h1>
  <h2>
    Not Found
  </h2>
</div>
`;
